jsapp/js/components/activity/formActivity.scss

```diff
@@ -0,0 +1,91 @@
+@use 'scss/sizes';
+@use 'scss/mixins';
+@use 'scss/breakpoints';
+@use 'scss/colors';
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: colors.$kobo-white !important;
+
+  .header {
+    width: 100%;
+    display: flex;
+    gap: sizes.$x8;
+    align-items: center;
+    padding: 0 sizes.$x20;
+
+    h1 {
+      color: colors.$kobo-gray-800;
+    }
+
+    .header-actions {
+      flex: 1;
+      display: flex;
+      gap: sizes.$x8;
+      justify-content: flex-end;
+    }
+
+    .filter-select {
+      width: sizes.$x150;
+    }
+  }
+
+  .table-container {
+    flex: 1;
+    overflow: auto;
+    padding: sizes.$x20;
+
+    .loading {
```

Comment:
There is no `.loading` anywhere, should we add it or delete this? I think it's something I've omitted in `UniversalTable`. It's a good moment to add it :) From my conversation with Tessa the loading indicator should be similar to Data Table one: 
![next-page-loader-action](https://github.com/user-attachments/assets/4a7b7b72-6076-43d7-86db-0c85ed387a3d)
There is an existing `LoadingSpinner` component that we should use here.

Reply:
Oh...I had that in my first implementation, using the class to display the loading spinner in the place of the universal table during the loading. Looks like I forgot to remove the class. Sorry about that.

Reply:
I cleaned up the CSS and I'm creating a task for us to add it to the UniversalTable